config/site.php

```diff
@@ -137,6 +137,11 @@
                         'options' => ['target' => '_blank'],
                         'title' => 'Slack',
                     ],
+                    'discord' => [
+                        'url' => 'https://discord.gg/A2VHpxeq',
```

Comment:
```suggestion
                        'url' => 'https://discord.gg/k4trEMPebj',
```

Changed temp invite link to non-expiring link.